apps/buildog/src/components/app-bar.tsx

```diff
@@ -1,8 +1,10 @@
 "use client";
 
 import React from "react";
+import { useRouter } from "next/navigation";
 import Link from "next/link";
 import { Avatar, AvatarFallback, AvatarImage } from "@ui/components/ui/avatar";
+import { DashboardIcon } from "@ui/components/ui/react-icons"; // Importing the Home icon from your icon library
```

Comment:
We don't need a comment here.
```suggestion
import { DashboardIcon } from "@ui/components/ui/react-icons";
```